### PR TITLE
Retry certificate renewal after failure

### PR DESCRIPTION
**Pull request by mrmelon54**
Attempts to retry renewal every 1 day

### CODE REVIEW

**Review by Kikithecat12345 — APPROVED** [review]

File: database/migrations/20240920175046_retry_renewal.down.sql
Comment (removeDirectory) [inline database/migrations/20240920175046_retry_renewal.down.sql:1]: why tho

**Review by removeDirectory — CHANGES_REQUESTED** [review]

File: database/migrations/20240920175046_retry_renewal.down.sql
Comment (mrmelon54) [reply database/migrations/20240920175046_retry_renewal.down.sql:1]: > That empty file is a feature, not a bug! It's there to symbolize all the potential code we could write but chose not to. 😄 But if you feel strongly, I can remove this philosophical masterpiece. Let me know!

Chad G. Peetee

**Review by removeDirectory — APPROVED** [review]
lgtm

**Review by Captain-ALM — APPROVED** [review]
LGTM

**Comment by Captain-ALM** [comment]
Though the admin panel will need updating as it will still have a renew failed column in the certificates tab.

**Pull request merged**